api/v1/controllers/aspects.js

```diff
@@ -187,6 +191,34 @@ module.exports = {
   },
 
   /**
+   * DELETE /aspects/{keys}/writers
+   *
+   * Deletes all the writers associated with this resource.
+   *
+   * @param {IncomingMessage} req - The request object
+   * @param {ServerResponse} res - The response object
+   * @param {Function} next - The next middleware function in the stack
+   */
+  deleteAspectWriters(req, res, next) {
+    doDeleteAllAssoc(req, res, next, helper, helper.belongsToManyAssoc.users);
+  },
```

Comment:
@shriramshankar is there a reason why we need to pass both helper and helper.belongsToManyAssoc.users?